journals/apps/journals/blocks.py

```diff
@@ -16,9 +18,17 @@
 STREAM_DATA_TYPE_FIELD = 'type'
 STREAM_DATA_DOC_FIELD = 'doc'
 
+SPAN_ID_FORMATTER = '{doc_type}-{doc_id}'
```

Comment:
This can be changed to 
`BLOCK_SPAN_ID_FORMATTER = '{block_type}-{block_id}'`

Reply:
yes, Going to change